README.md

```diff
@@ -135,6 +136,24 @@ another reference bound to the same property, it unbinds it first.
 }
 ```
 
+```js
+bindFirebaseRef('todos', ref, {
+  cancelCallback: (cancel) => {
+    // :TODOS
+    // cancel is a value of cancelCallback
```

Comment:
These should contain more real world examples 😄 
Also, could you remove the TODOS comment?